phase2/kubeadm/configure-vm-kubeadm-master.sh

```diff
@@ -37,7 +37,8 @@ if [[ "$KUBEPROXY_MODE" == "ipvs" ]]; then
     cat <<EOF |tee -a $KUBEADM_CONFIG_FILE
 kubeProxy:
   config:
-    featureGates: SupportIPVSProxyMode=true
+    featureGates:
+      SupportIPVSProxyMode=true
```

Comment:
Should maybe be `SupportIPVSProxyMode: true`

Reply:
Right! I'll change that.